src/lib/utils/thorchain/index.ts

```diff
@@ -211,16 +212,45 @@ export const getAccountAddresses = memoize(
     (await getAccountAddressesWithBalances(accountId)).map(({ address }) => address),
 )
 
+// Gets all pools, including unavailable
+export const getThorchainPools = async (): Promise<ThornodePoolResponse[]> => {
+  const queryKey = ['thornodePoolData']
+  const queryFn = async () => {
+    const { data } = await axios.get<ThornodePoolResponse[]>(
+      `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/pools`,
+    )
+
+    return data
+  }
+
+  const opportunitiesData = await queryClient.fetchQuery({
+    queryKey,
+    queryFn,
+    staleTime: Infinity,
+  })
+
+  return opportunitiesData
+}
+// WARNING: This uses Infinity staleTime, so will never refetch unless the query is invalidated
+// Consume with caution in domains other than swapper
+// Swapper does *not* use a cached react-query, and uses the thorService variant which has its own caching mechanism
 export const getThorchainAvailablePools = async () => {
   const daemonUrl = getConfig().REACT_APP_THORCHAIN_NODE_URL
-  const poolResponse = await thorService.get<ThornodePoolResponse[]>(
-    `${daemonUrl}/lcd/thorchain/pools`,
-  )
-  if (poolResponse.isOk()) {
-    const allPools = poolResponse.unwrap().data
-    const availablePools = allPools.filter(pool => pool.status === 'Available')
-    return availablePools
+  const queryKey = ['thornodePoolData']
```

Comment:
Pretty much what the comment says - this is used in savers and lending, which will all use the same react-query caching.

This is medium to high risk as we may never fetch new data for savers, and will need to be thoroughly tested.

Reply:
Both dangerous *and* superfluous since all consumptions are react-query queries themselves - reverted in d1ff4fa8fe7df246c1c82dc0c020f2c127416c78